service/history/mutableStateBuilder.go

```diff
@@ -169,6 +169,22 @@ func newMutableStateBuilderWithReplicationState(config *Config, logger bark.Logg
 	return s
 }
 
+func (e *mutableStateBuilder) Unload() *persistence.WorkflowMutableState {
```

Comment:
The function name is unclear. Maybe name it as CopyTo...

Reply:
I felt so too. did that because we already have 
```
func (e *mutableStateBuilder) Load(state *persistence.WorkflowMutableState)" doing the opposite. 
```
CopyTo might not be a better name because mutableStateBuilder and mutabeState struct are not exactly the same.
To be more clear, we probably can call it loadFromMutableState and unloadToMutableState.

Reply:
Unload give false impression that you are resetting the mutable state.

Reply:
Make sense. How about rename it to "copyTo"?